types/react-leaflet/index.d.ts

```diff
@@ -196,13 +196,13 @@ export interface TileLayerProps extends TileLayerEvents, Leaflet.TileLayerOption
     children?: Children;
     url: string;
 }
-export class TileLayer<P extends TileLayerProps, E extends Leaflet.TileLayer> extends GridLayer<P, E> { }
+export class TileLayer<P = TileLayerProps> extends GridLayer<P, Leaflet.TileLayer> { }
```

Comment:
Can we keep both type parameters here? So:
```typescript
export class TileLayer<P = TileLayerProps, E = Leaflet.TileLayer> extends GridLayer<P, E> { }
```
Same issue on lines: 205, 323, 351